server/middleware/fetchLicenceMiddleware.test.ts

```diff
@@ -5,39 +5,109 @@ import { Licence } from '../@types/licenceApiClientTypes'
 
 jest.mock('../services/licenceService')
 
-const req = {} as Request
-const res = { locals: { user: { username: '' } } } as unknown as Response
+let res = {} as unknown as Response
+const req = {
+  path: '/licence/create/id/1/check-your-answers',
+} as Request
 const next = jest.fn()
+
+const prisonRes = {
+  locals: {
+    user: {
+      username: 'XX',
+      nomisStaffId: 123,
+      prisonCaseload: ['LEI', 'MDI'],
+    },
+    licence: undefined,
```

Comment:
Dont need it here if its undefined?

Reply:
I think its needed, but can check again.

Reply:
Tests alter the object by populating a licence in it.